fuzz/sst_file_writer_fuzzer.cc

```diff
@@ -62,27 +91,31 @@ DEFINE_PROTO_FUZZER(DBOperations& input) {
   rocksdb::SstFileWriter writer(env_options, options);
   rocksdb::Status s = writer.Open(sstfile);
   CHECK_OK(s);
+  std::map<std::string, std::string> kv;
```

Comment:
SstFileWriter doesn't sort data, so it mirrors to a vector. Why map?

Reply:
Never mind. I was wrong.

In that case, maybe SstFileReader isn't the best interface to do the validation. We need something lower level. Something like directly returning from TableReader.

Reply:
I agree. I would rather see a "round trip" of the data where you can compare it with `input.operations()` again; no need for a 'kv' map or vector.